build.sbt

```diff
@@ -33,7 +45,7 @@ lazy val scalacheck = project
       "org.scalacheck" %% "scalacheck" % "1.14.0"
     )
   )
-  .dependsOn(core, `test-commons` % "test->test")
+  .dependsOn(core, generic, `test-commons` % "test->test")
```

Comment:
I'd rather not make `scalacheck` (or any other module) depend on `generic` if possible (it feels like it breaks modularity).

That of course would prevent us to provide a `GenModule` based on `generic`, but I don't think it's very problematic (and a generic version of `GenModule` can still be implemented in `generic`'s tests).

That point remains open to discussion though.